.clang-format

```diff
@@ -3,3 +3,11 @@ ColumnLimit: 90
 DerivePointerAlignment: false
 IndentCaseLabels: false
 PointerAlignment: Right
+BinPackArguments: false
+BinPackParameters: false
+AllowAllArgumentsOnNextLine: false
+AllowAllParametersOfDeclarationOnNextLine: false
+InsertTrailingCommas: Wrapped
+AlignAfterOpenBracket: AlwaysBreak
+AllowAllConstructorInitializersOnNextLine: true
```

Comment:
Is this consistent, with `AllowAllConstructorInitializersOnNextLine` to `true` and other `*OnNextLine` settings `false`? `folly` seems to set `BinPack*` to false and `*OnNextLine` to true.